docs/source/onedal/build_app/build-application.rst

```diff
@@ -17,21 +17,81 @@
 Build applications with oneDAL
 ==============================
 
-See Get Started Guides for `Windows\*
-<https://software.intel.com/content/www/us/en/develop/documentation/get-started-with-daal-for-windows/top.html>`_
-and `Linux\* <https://software.intel.com/content/www/us/en/develop/documentation/get-started-with-daal-for-linux/top.html>`_
-for instruction on how to build applications for C++.
 This section contains instructions for building applications with |short_name| for SYCL\*.
 
-- :ref:`app_on_win`
```

Comment:
what about mac?

Reply:
well I only restructured what we had, and we didn't have anything for mac.
I'd suggest adding instructions for mac in a separate PR